source/core/NumericString.cpp

```diff
@@ -2593,15 +2646,28 @@ void ScanSpreadsheet(StringRef inputString, StringRef formatString, StringRef de
     }
 }
 //-------------------------------------------------------------------
-VIREO_FUNCTION_SIGNATURE4(SpreadsheetStringtoArray, StringRef, StringRef, StringRef, TypedArrayCoreRef)
+VIREO_FUNCTION_SIGNATURE4(SpreadsheetStringtoArray, StringRef, StringRef, TypedArrayCoreRef, TypedArrayCoreRef)
+{
+    TypedArrayCoreRef outputArray = _Param(3);
+    if (_Param(2)->Length() ==0) {
```

Comment:
I think this code can be further simplified by setting delimiter to \t or _Param(2).

Reply:
This isn't really an option because it would rely on construction of the STACK_VAR variable even in the case where it wouldn't be used.  Currently it's only used in the block where _Param(2) is zero length.